source/configuring_domain/02_managing_legal_information/legals.rst

```diff
@@ -68,22 +68,68 @@ To specify the Terms and Conditions and the Privacy Policy of a portal, go to Co
 Licenses
 --------
 
-When configuring the metadata of a dataset in the Information tab, the license attached to that dataset should be defined, using a drop-down selection of predefined licenses. To define the licenses listed in the drop-down selection, go to Configuration > Legals. In this subsection of the back office menu, look for the "Licenses" area.
+When configuring the metadata for a dataset in the Information tab, it is important to define the license that specifies how others can use the data.
```

Comment:
(suggestions) 
+ maybe add that re-users (external developers, data analysts…) are more likely to work with properly licensed data, rather than go out of their way to contact the producer and ask them more information?
+ ➕ giving a known license also reduces the potential bad buzz on social media, wherever a community of hardcore open-source promoters exists (my formulation is not the diplomatic one, but I’ll be happy to discuss it if needed)

Reply:
Great suggestions!
The intro could be edited to include more info about the need for choosing the right licensing option:

When configuring the metadata for a dataset in the Information tab, it is important to define the license that explicitly specifies how others can use the data.
Reusers - external developers, data analysts, etc. - are more likely to work with properly licensed data than contacting data producers and ask them for more information.
Work with your legal team to find the most appropriate license for your data, depending on your objectives.

Reply:
your wording is far better than mine 😅